test/e2e/network/kube_proxy.go

```diff
@@ -171,16 +171,16 @@ var _ = SIGDescribe("Network", func() {
 		// If test flakes occur here, then this check should be performed
 		// in a loop as there may be a race with the client connecting.
 		framework.IssueSSHCommandWithResult(
-			fmt.Sprintf("sudo cat /proc/net/ip_conntrack | grep 'dport=%v'",
+			fmt.Sprintf("sudo cat /proc/net/nf_conntrack | grep 'dport=%v'",
 				testDaemonTcpPort),
 			framework.TestContext.Provider,
 			clientNodeInfo.node)
 
 		// Timeout in seconds is available as the third column from
```

Comment:
I believe it's the _fifth_ column of `nf_conntrack` (third for `ip_conntrack` - output is slightly different, since nf_conntrack has two extra columns at the beginning).

Reply:
according the https://stackoverflow.com/questions/16034698/details-of-proc-net-ip-conntrack-nf-conntrack, thank you for pointing that out to me